app/javascript/plugins/fundraiser/FundraiserView.js

```diff
@@ -21,7 +23,28 @@ import {
 } from '../../state/fundraiser/actions';
 
 export class FundraiserView extends Component {
-  componentDidMount() {
+  async componentDidMount() {
+    if (window.dataLayer) {
+      await window.dataLayer.push({
+        event: unintendedDonationsExperiment.activationEvent,
+      });
+    }
+    this.intervalId = setInterval(() => {
+      if (window.google_optimize !== undefined) {
+        const variant = window.google_optimize.get(
+          unintendedDonationsExperiment.experimentId
+        );
+        console.log('variant', variant);
```

Comment:
I added this for debugging/testing purposes, and I will remove it in a follow-up PR after testing it on staging.